### ISSUE TITLE
3D Building Support gradient-color

### ISSUE THREAD

**Issue by dythbj** [mapbox/mapbox-gl-js#6968]
In 'fill-extrusion' layer,mapbox-gl-js only support 'fill-extrusion-color' for a single color,is there any way that we can render the 3d building like this:
![wx20180716-153024](https://user-images.githubusercontent.com/41282663/42747285-658584f0-890e-11e8-95e2-ff204a5d2f31.png)
Hope for you help!

**Comment by mourner** [comment]
No, but you can get a similar look by adjusting the global light properties: https://www.mapbox.com/mapbox-gl-js/style-spec#light

**Comment by dythbj** [comment]
> No, but you can get a similar look by adjusting the global light properties: https://www.mapbox.com/mapbox-gl-js/style-spec#light

Thanks mourner ,but the light is not i want，especially irregular rendering in the vertical direction

**Comment by blackMoja** [comment]
Is this function supported now? 👀

**Comment by dythbj** [comment]
> Is this function supported now? 👀

Nope

**Comment by blackMoja** [comment]
@dythbj  Then, can I understand that gradation is not possible for 3D buildings yet? 👀

**Comment by AnasArafeh** [comment]
@mourner @underoot I have seen that this issue was marked as feature then was closed in Same day, has this been developed or closed by mistake ? as this feature is good to have

**Comment by underoot** [comment]
@AnasArafeh It was closed because a workaround was provided. But to reflect the issue's actual status, I will re-open it. Thank you!

**Comment by AnasArafeh** [comment]
@underoot Do you have any clear timeline for when this can be supported ?

**Comment by underoot** [comment]
@AnasArafeh Not yet
